pkg/integration/tests/branch/select_commits_of_current_branch.go

```diff
@@ -0,0 +1,74 @@
+package branch
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var SelectCommitsOfCurrentBranch = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Select all commits of the current branch",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
```

Comment:
I'm a little confused by this test: is there a merge commit here that I can't see?